Riot/Modules/Room/RoomInfo/RoomInfoList/RoomInfoListViewController.swift

```diff
@@ -200,18 +200,12 @@ final class RoomInfoListViewController: UIViewController {
         if RiotSettings.shared.roomInfoScreenShowIntegrations {
             rows.append(rowIntegrations)
         }
-<<<<<<< HEAD
-        if !isRoomDirect {
```

Comment:
@giomfo do you know if we need to keep this change for Tchap ?

Reply:
Done